index.html

```diff
@@ -51,17 +70,17 @@ <h2>Pick Your Destination</h2>
       <div class="destination">
         <h4>Fun In The Sun</h4>
         <p>Expedition excursion design excursion fun, clean simple organized WordPress Travel colorful webdesign. Traveler blogger website design expedition clean excursion traveling.</p>
-        <div class="btn">Sign Me Up!</div>
+        <button><div class="btn">Sign Me Up!</div></button>
```

Comment:
When you create a button, you don't have to use `<div>` inside of it. This button could look like this: ```<button class="btn">Sign Me Up!</button>```